Automattic/newspack-plugin
# fix: localized reader auth error messages

### All Submissions:






### Changes proposed in this Pull Request:

This PR localizes the in-script error messages using `wp_localize_script()`. This strategy was chosen over `@wordpress/i18n` to not impact the number of frontend dependencies.

It also introduces an error message for a failed OAuth popup.


### How to test the changes in this Pull Request:

1. Check out this branch and with active RAS, open the Sign In modal
2. Attempt to sign in without an email and confirm you receive the proper error message
3. On password mode, attempt to sign in without a password and confirm the message

#### Blocked popup

1. Open the site in Firefox with the [Popup Blocker Ultimate](https://addons.mozilla.org/pt-BR/firefox/addon/popup-blocker-ultimate/) extension
2. Click the Sign In modal and attempt to authenticate using Google
3. Confirm the popup is blocked and the error message is thrown
4. Allow popups, try again and confirm you are able to authenticate

### Other information:
